src/main/java/ReviewResponse.java

```diff
@@ -2,6 +2,8 @@
 import lombok.Getter;
 import lombok.ToString;
 
+import java.util.List;
+
 @ToString
 @Data
 public class ReviewResponse {
```

Comment:
This method violates the Single Responsibility Principle (SRP) as it combines two different responsibilities: comparing strings and returning a boolean. It would be better to split this method into two separate methods for better code organization. (Severity: medium)